src/web-components/comments/components/content.ts

```diff
@@ -30,7 +30,21 @@ export class CommentsContent extends WebComponentsBaseElement {
     annotationFilter: { type: String },
   };
 
+  private unselectAnnotation = () => {
+    this.selectedAnnotation = null;
+  };
+
+  private unselectAnnotationEsc = (event?: KeyboardEvent) => {
+    if (event && event?.key !== 'Escape') return;
+    this.selectedAnnotation = null;
+  };
+
   private selectAnnotation = ({ detail }: CustomEvent) => {
+    if (this.selectedAnnotation === detail.uuid) {
+      this.selectedAnnotation = null;
+      return;
+    }
+
     const { uuid } = detail;
     this.selectedAnnotation = uuid;
   };
```

Comment:
```suggestion
  private selectAnnotation = ({ detail }: CustomEvent) => {
      const { uuid } = detail;
  
    if (this.selectedAnnotation === uuid) {
      this.selectedAnnotation = null;
      return;
    }

    this.selectedAnnotation = uuid;
  };
```

Also add JS docs to this method